pkg/needacert/needacert.go

```diff
@@ -218,7 +226,12 @@ func (h *handler) OnCRDChange(key string, crd *apiextv1.CustomResourceDefinition
 
 	service, err := h.serviceCache.Get(crd.Spec.Conversion.Webhook.ClientConfig.Service.Namespace,
 		crd.Spec.Conversion.Webhook.ClientConfig.Service.Name)
-	if err != nil {
+
+	if apierror.IsNotFound(err) {
+		// OnService will be called when the service is created, which will eventually update the CRD, so no
```

Comment:
~Shouldn't this be `OnCRDChange` ? OnService doesn't appear to call this.~ Looks like it does enqueue crd's